client/js/recurring_donations/edit/change-amount-wizard.es6

```diff
@@ -3,7 +3,6 @@ const flyd = require('flyd')
 const R = require('ramda')
 const h = require('snabbdom/h')
 const url = require('url')
-const render = require('ff-core/render')
```

Comment:
Is this related to removing ramda? If not, let's do as separate PR.

Reply:
@wwahammy I can split these out yep. There are a few of them so far I think, just removing unused import lines as I come across them. So cleanup related, but not ramda-specific.